src/build/app/mod.rs

```diff
@@ -1536,7 +1545,7 @@ impl<'b> App<'b> {
         true
     }
 
-    pub fn _propagate(&mut self, prop: Propagation) {
+    pub(crate) fn _propagate(&mut self, prop: Propagation) {
```

Comment:
This is not related to this PR.

Reply:
All of `pub => pub(crate)` are directly related to this PR.

The point of the PR is to separate things into three different categories:
* Items that are not needed outside of the module, they don't need `pub` at all.
* Items that are needed somewhere else in clap, but must not be public. Those get `pub(crate)`.
* The API exposed to users - `pub` (no hidden!).

Reply:
Yeah, related to the PR. But I want us to separate the view stuff out in this PR.

Reply:
But how do you expect me to do that? "The view stuff" is what makes it possible to move from `pub` to `pub(crate)`.

Reply:
I can delay it until the view is merged, but honestly, I don't see the point. Should I need to do that?